### ISSUE TITLE
rename package?

### ISSUE THREAD

**Issue by lazarusA** [pxl-th/Trace.jl#31]
Maybe, after all Simon's updates, it would be good to give an extra bust to this package with a new name? Just an idea and not suggestions at the moment 😓 .

**Comment by SimonDanisch** [comment]
Yeah we need to, since Trace.jl is already a registered package ;)
